docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to proxy peering mode.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  TLS routing is available starting from Teleport `10.0`.
+</Details>
+
+## Requirements
+
+Teleport Proxy servers must be able to reach each other over the network. Ensure
+there are no firewall policies that would block proxy to proxy communication.
+
+## Step 1/4. Upgrade to teleport `10.0`
+Download Teleport `10.0` or later from the [downloads page](https://goteleport.com/teleport/download) or your enterprise portal
+and follow the standard [upgrade procedure](https://goteleport.com/docs/setup/operations/upgrading/).
+
+## Step 2/4. Enable proxy peering
+Update your cluster's auth server configuration to set the tunnel strategy type to `proxy_peering`
+
+```yaml
+auth_service:
+    tunnel_strategy:
+        type: proxy_peering
+        agent_connection_count: 1
+```
+
+This setting will indicate to agents that they are only required to connect to 1 proxy server as specified by the
+`agent_connection_count` field.
+
+## Step 3/4. Restart auth services
+Restart all auth services running in the cluster to apply the new auth service configuration
```

Comment:
What's the best way to do this? If you have multiple Auth Service instances, can you restart them in any order?

Reply:
What do you mean by best way to do this? My initial reaction is it depends how teleport is deployed. If there are multiple auth services the order doesn't matter as long as the configs are the same.